cumulusci/utils/tests/test_git.py

```diff
@@ -69,3 +71,8 @@ def test_construct_release_branch_name():
 def test_parse_repo_url(repo_uri, owner, repo_name, host):
     assert parse_repo_url(repo_uri) == (owner, repo_name, host)
     assert split_repo_url(repo_uri) == (owner, repo_name)
+
+
+def test_empty_url():
+    with pytest.raises(error, match="Url is none or must have `remote` set as origin"):
```

Comment:
Use the more specific exception (`ValueError` if changed as per suggestion above) instead of the generic `error`.